docs/Comparison.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: "EL8 Distro Comparison"
+title: "EL Distro Comparison"
 ---
 
 | Benchmarking against RHEL                  | AlmaLinux                | Oracle Linux                                   | Rocky Linux                                 | CentOS Stream     | CentOS Linux      |
 | -------------------------------------------|--------------------------|------------------------------------------------|-------------------------------------------- |-------------------|-------------------|
 |Production Version                          | Since March 2021         | Since 2006                                     | Since June 2021                             | Since 2019        | Since 2004        |
-|Strives 1:1 binary RHEL binary compatibility| Yes                      | Yes                                            | Yes                          | Compatible within the limits of the [ACG][acg] | Yes |
+|Strives 1:1 binary RHEL binary compatibility| ABI Compatible           | Yes                                            | Yes                          | Compatible within the limits of the [ACG][acg] | Yes |
```

Comment:
```suggestion
|RHEL compatibility| ABI Compatible           | [ABI Compatible?](https://www.oracle.com/news/announcement/blog/keep-linux-open-and-free-2023-07-10/)                                            | Binary compatible                          | Compatible within the limits of the [ACG][acg] | Binary Compatible |
```

Reply:
I'd rather change the title of this metric now since it's getting a bit more complex than just "are they 1:1".